template_feed/Microsoft.DotNet.Common.ItemTemplates/content/Solution/Solution1.sln

```diff
@@ -1,8 +1,8 @@
-﻿
+
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.6.30114.105
-MinimumVisualStudioVersion = 10.0.40219.1
+# Visual Studio Version 15
+VisualStudioVersion = 15.0.26124.0
+MinimumVisualStudioVersion = 15.0.26124.0
```

Comment:
@marcpopMSFT 

Pardon, could you explain what is the technical reason of using version number of 10.0.xxx.xxx? What about the previous behavior in .NET Core 3.0 and 3.1 SDKs?
I mean, as far as I know, all of the solution generated by .NET Core 3.0.xxx, and also in Core 3.1.101 to 3.1.407 SDK always have minimum version set to `15.0.26124.0`, for example, this is the SLN generated from 3.1.406 SDK:

```
Microsoft Visual Studio Solution File, Format Version 12.00
# Visual Studio 15
VisualStudioVersion = 15.0.26124.0
MinimumVisualStudioVersion = 15.0.26124.0
Global
        GlobalSection(SolutionConfigurationPlatforms) = preSolution
                Debug|Any CPU = Debug|Any CPU
                Debug|x64 = Debug|x64
                Debug|x86 = Debug|x86
                Release|Any CPU = Release|Any CPU
                Release|x64 = Release|x64
                Release|x86 = Release|x86
        EndGlobalSection
        GlobalSection(SolutionProperties) = preSolution
                HideSolutionNode = FALSE
        EndGlobalSection
EndGlobal
```

But if this change of versioning behavior from previous Core 3.0 and 3.1 is intended and should be enforced, then I will change this to use version `10.0.40219.1` instead. Please confirm and thanks! 🙂

Reply:
I'd try to have dotnet match what VS does which is target 10.  It's possible 3.0 picked 15 since .net core didn't exist back in 10.0.  Ultimately, I'll leave this to @donJoseLuis as the owner of the templating repo to make the call on what he wants the values to be.

Reply:
@marcpopMSFT 

Ah, you're absolutely right! I have checked, and the `MinimumVisualStudioVersion` from VS 2019 is `10.0.40219.1`

I have updated this MinimumVisualStudioVersion` to use the 10.0.40219.1` version number.
Please review.

I also welcome @donJoseLuis to review 🙂